### ISSUE TITLE
Dynamic Resizing

### ISSUE THREAD

**Issue by sashaperigo** [notwaldorf/tiny-care-terminal#109]
This is definitely a nice-to-have, but it would be great if Tiny Care Terminal resized dynamically when you changed the size of your Terminal window!

- **Node Version**: v8.1.2
- **tiny-core-terminal version**: 1.3.1
- **Shell**: bash
- **Terminal Program**: cmd? I have no idea.
- **Operating System**: Linux

**Comment by victorperin** [comment]
I would love that too! :)

**Comment by rareinator** [comment]
@mojoaxel How does it look for you. Cause for me the emojis loose some space after them. So everything is moved a bit to the right.
![screen shot 2018-03-12 at 00 09 42](https://user-images.githubusercontent.com/5727398/37259698-e56fb110-2589-11e8-9868-cb7b7788e678.png)

**Comment by steoneill** [comment]
@rareinator I'm seeing this too.

**Comment by mojoaxel** [comment]
If the window gets TOO small I also get problems with the commity box:
![image](https://user-images.githubusercontent.com/600565/37389969-9429fdae-2766-11e8-9a15-b032985c5c35.png)

**Comment by rareinator** [comment]
@mojoaxel Yeah but even when it's in full screen, it seems that the emojis aren't being displayed properly. I feel like there is not enough room on the right side. Again this is an issue with hyper. I would just love to know how you got it to work.  😀

**Comment by mojoaxel** [comment]
>  it seems that the emojis aren't being displayed properly

If you have problems with emojis please see #92 and #44

**Comment by rareinator** [comment]
@mojoaxel Yeah tried that but it didn't work. But if I update hyper to the newest canary release. It works just fine. So might be something the hyper team already fixed.
